src/utils/formatSearchResponse.ts

```diff
@@ -48,6 +55,17 @@ const formatSearchResponse = async (
       organisationName: organisationDetails.organisationValue,
     };
 
+    if (isMapResults) {
+      const coordinatesData: IAccumulatedCoordinatesWithCenter | null = getAccumulatedCoordinatesNCenter(
+        searchItem?._source?.geom ?? {},
+      );
+      item = {
+        ...item,
+        geographicBoundary: coordinatesData?.coordinates,
+        geographicCenter: coordinatesData?.center,
```

Comment:
geographicCenter & geographicBoundary value may be undefined. but at location.js line 277 you are using it. location.js works with undefined values?

Reply:
For map results, we use an ES query to identify the record that contains geocoordinate data. There is no possibility of undefined values for coordinates and center.